### DISCUSSION TITLE
谢谢

### DISCUSSION THREAD

**Discussion by muxikuku** [pjy612/SteamManifestCache#718] [discussion]
谢谢各位大佬们的无私奉献~同时作为内测用户也打赏表明了一点心意，加油！！

**Comment by pjy612** [comment]
emmm Free 实际不算内测...

**Reply by muxikuku** [reply]
> emmm Free 实际不算内测...

我的意思是我去年就拿了内测那个文件啦，只是当时停止维护了TT

**Reply by pjy612** [reply]
记不清了，一般 有 分享游戏 或 账号 的 才会拉

**Reply by coco2631** [reply]
> 记不清了，一般 有 分享游戏 或 账号 的 才会拉

大佬为什么会出现Bad credentials啊，有解决的办法吗

**Reply by muxikuku** [reply]
> 记不清了，一般 有 分享游戏 或 账号 的 才会拉

是拉群吗还是什么 我没有被拉群，如果有群很高兴被拉进去。然后分享账号要怎么做，我号里有176个游戏都可以分享。

**Reply by muxikuku** [reply]
> 记不清了，一般 有 分享游戏 或 账号 的 才会拉

我只有SAI.lic

**Reply by pjy612** [reply]
> > 记不清了，一般 有 分享游戏 或 账号 的 才会拉
> 
> 是拉群吗还是什么 我没有被拉群，如果有群很高兴被拉进去。然后分享账号要怎么做，我号里有176个游戏都可以分享。

如果你有过 lic 的话，应该有邮件，之后 邮件 或 tg 沟通。

**Reply by muxikuku** [reply]
> > > 记不清了，一般 有 分享游戏 或 账号 的 才会拉
> > 
> > 
> > 是拉群吗还是什么 我没有被拉群，如果有群很高兴被拉进去。然后分享账号要怎么做，我号里有176个游戏都可以分享。
> 
> 如果你有过 lic 的话，应该有邮件，之后 邮件 或 tg 沟通。

您好！发过邮件了，两封石沉大海(,,,
